hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/util/ClientIds.java

```diff
@@ -69,7 +70,7 @@ public class ClientIds implements AutoCloseable, Serializable {
 
   private static final String HEARTBEAT_FOLDER_NAME = ".ids";
   private static final String HEARTBEAT_FILE_NAME_PREFIX = "_";
-  public static final String INIT_CLIENT_ID = "";
+  public static final String INIT_CLIENT_ID = String.valueOf(RandomUtils.nextInt(0,100));
   public static final long DEFAULT_HEARTBEAT_INTERVAL_IN_MS = 60 * 1000; // default 1 minute
```

Comment:
Can we use Java util `Random` instead commons lang3, the apache commons is likely to conflict.

Reply:
> Can we use Java util `Random` instead commons lang3, the apache commons is likely to conflict.

OK